### DISCUSSION TITLE
Share your config here!

### DISCUSSION THREAD

**Discussion by tehniemer** [mza921/Plex-Auto-Collections#161] [Show and tell]
@ramair02 brought up the idea in #140 and I just noticed discussions were open so I figure I'd kick this off. Here's mine.

[config.txt](https://github.com/mza921/Plex-Auto-Collections/files/5761116/config.txt)

**Comment by ramair02** [comment]
Very cool! Here's my latest
[config-ramair02.yml.template.zip](https://github.com/mza921/Plex-Auto-Collections/files/5760655/config-ramair02.yml.template.zip)

**Reply by tehniemer** [reply]
Thanks, where did you pull the summaries for your studio collections? They are nice and concise, I like them.

**Reply by ramair02** [reply]
Most are the first paragraph from Wikipedia.

I added some of your collections to my config, so thank you! Much easier than keeping a TMDB window open and searching for each collection ID. Hoping others share their configs and can also benefit from the work we've put in. Maybe share this on r/plex subreddit?

**Comment by AwesomeAustn** [comment]
@tehniemer I used some of your lists and formatted mine like yours with the sections. Thanks!
https://mega.nz/folder/f0UWSbaY#_pcsCG-GUoSg_KUPFP8dXw

Edit: Added my Movie and TV Show configs to MEGA, along with my custom posters and PSDs. I used Coverlabs' Ripple Design (https://www.coverlabs.io/design/ripples) for the backgrounds and text, and Flaticon (https://www.flaticon.com) for the icons. Using MEGA Sync helps keep everything up to date without re-uploading them here.
https://mega.nz/folder/D8tgyBxb#1U8CmT9G7eG04X8u6J2pbQ

**Comment by ramair02** [comment]
@tehniemer I like the formatting in the script. Does this affect how Collections are displayed at all? Or rather is it only for ease of organizing the yml file?

It would cool to be able to add some sort of page break / divider. I'd do something like DECADES // TOP LISTS // STUDIOS // PEOPLE // SEQUELS.

Here are how my Collections look at the moment: https://i.ibb.co/cFCtV27/Plex-Collections.png

**Reply by tehniemer** [reply]
It doesn't affect anything the script does, I just find it easier to read with some formatting.

**Comment by BlueSkillZ99** [comment]
Here is my work in progress status, including movies and shows

https://gist.github.com/BlueSkillZ99/57c6f116191a38aab287a07d3b3546f6

**Comment by ramair02** [comment]
Here is my latest with some changes and new Best of Genre Collections

[config.yml.ramair02.yml.zip](https://github.com/mza921/Plex-Auto-Collections/files/5810852/config.yml.ramair02.yml.zip)

**Reply by BlueSkillZ99** [reply]
Nice work. Does script work with sonarr as well, thought only radarr?

In your genre searches, I would add a minimum vote count limit. Just to make it faster. Look e.g at your action search, where 300 out of 500 movies have less than 100 votes. Especially on Action I use a count >100,000

**Comment by ramair02** [comment]
Whoops! Copied & Pasted my movie config into my tv config template. (Only difference in my 'template' is the removed tokens, api keys, etc.)
Here is movies: [config.yml.ramair02.yml.zip](https://github.com/mza921/Plex-Auto-Collections/files/5811962/config.yml.ramair02.yml.zip)
Here is tv:
[config.yml.ramair02.yml.zip](https://github.com/mza921/Plex-Auto-Collections/files/5811976/config.yml.ramair02.yml.zip)

No, it doesn't work with Sonarr -- you are correct. I had tried it before doing some research and learning that it doesn't work with Sonarr. I had left it in the config anyway hoping that one day soon it will work :)

I don't think I've seen anything in the README about minimum vote count, but I suppose you're suggesting add it as a criteria for IMDB search. Do I have to do each search over or is there a string I can add to the URL? Great idea, thank you.

**Reply by BlueSkillZ99** [reply]
you just add something like `&num_votes=10000,`

like here: https://www.imdb.com/search/title/?title_type=feature&release_date=1990-01-01,&user_rating=5.0,10.0&num_votes=100000,&genres=action

**Reply by ramair02** [reply]
That worked great. Thank you.
